src/ui/Facet/Facet.ts

```diff
@@ -275,8 +277,16 @@ export class Facet extends Component {
      * option, or `occurrences` if no sort criteria is specified.
      */
     sortCriteria: ComponentOptions.buildStringOption({
-      postProcessing: (value, options: IFacetOptions) =>
-        value || (options.availableSorts.length > 0 ? options.availableSorts[0] : 'occurrences'),
+      postProcessing: (value, options: IFacetOptions) => {
+        const logger = new Logger(Facet);
+        const valueIfUndefined = value || (options.availableSorts.length > 0 ? options.availableSorts[0] : 'occurrences');
```

Comment:
`valueIfUndefined` 🤔...

I'd call it instead `sortCriteriaToValidate`

Reply:
True, I only had a problem when the value was undefined but now, it will be used for all the value so I'll change it